src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/DeclareTypesInNamespacesTests.cs

```diff
@@ -10,14 +10,97 @@ namespace Microsoft.ApiDesignGuidelines.Analyzers.UnitTests
 {
     public class DeclareTypesInNamespacesTests : DiagnosticAnalyzerTestBase
     {
+        private static readonly string s_ruleId = DeclareTypesInNamespacesAnalyzer.RuleId;
+        private static readonly string s_message = MicrosoftApiDesignGuidelinesAnalyzersResources.DeclareTypesInNamespacesMessage;
+
+        [Fact]
+        public void OuterTypeInGlobalNamespace_Warns()
```

Comment:
Huh. Now this is a nice convention that I haven't seen anybody else use: putting the C# and VB tests into the same test case. It does make it easy to see them together, and helps you not forget to implement the VB case for each test.